### ISSUE TITLE
Домофоны от Интерсвязи

### ISSUE THREAD

**Issue by VikeDragon** [alexmorbo/domru#35]
Добрый день!
Насколько возможно реализовать похожий проект для домофонов от провайдера Интерсвязь. Так же ставят домофоны, есть приложения для получения удаленно картинки и открытия?

Полное приложение https://play.google.com/store/apps/details?id=com.intersvyaz.lk&hl=ru
Урезанное приложение, только открывашка https://play.google.com/store/apps/details?id=net.intersvyaz.intercom&hl=ru

**Comment by alexmorbo** [comment]
Есть планы по созданию нового дополнения, в котором будет не только домру, но и [1net](https://1net.by/dlya-doma/videonablyudenie-i-domofoniya/) 

Можно и вашего оператора добавить.
Напишите в [телеграм](https://t.me/alexmorbo), оставьте контактные данные, я когда доберусь (надеюсь доберусь), то посмотрю и вашего оператора

**Comment by Crazy-Child** [comment]
Да, очень хотелось бы увидеть реализацию данного проекта для домофонов от провайдера Интерсвязь. Особенно домофонные панели у них идентичные, отличаются только лейблами фирмы.

**Comment by Vladsssssss** [comment]
Присоединяюсь, жду решения для интерсвязи

**Comment by xxxCAH4E3xxx** [comment]
+1, ждём для интерсвязи. даже просто live уже хорошо, а в идеале со звуком, кнопкой открытия и событием вызова)
